src/components/MovieForm/components/AddAndEditMovieDialog/AddAndEditMovieDialog.jsx

```diff
@@ -2,33 +2,47 @@ import React from "react";
 import Dialog from "../../../Dialog/Dialog";
 import MovieForm from "../../MovieForm";
 import { AddMovieText } from "../../../../helpers/constants";
-import PropTypes from "prop-types";
+import { useNavigate, useOutletContext } from "react-router-dom";
+import { createMovie, updateMovie } from "../../../../services";
 
-const AddAndEditMovieDialog = ({ isOpen, onClose, movieToEdit }) => {
-  
-  const handleFormSubmit = () => {
-    onClose();
+const AddAndEditMovieDialog = () => {
+  const { movie } = useOutletContext();
+  const navigate = useNavigate();
+
+  const closeForm = () => {
+    navigate(-1);
```

Comment:
Notice - after submitting changes, the data on your movie isn't updated.
1. Open movie
2. Edit movie
3. Change title
4. Submit

The title isn't changes. Have to reload page to see the update.

Reply:
resolved